src/build.rs

```diff
@@ -56,69 +56,77 @@ impl Default for BuildOptions {
 impl Parse for BuildOptions {
     fn parse(input: ParseStream) -> Result<Self> {
         let mut out = Self::default();
+
         while !input.is_empty() {
-            let key = input.parse::<Ident>()?;
-            match &key.to_string()[..] {
-                "kind" => {
-                    input.parse::<Token![:]>()?;
-
-                    let value = input.parse::<Ident>()?;
-                    if let Some(kind) = extension_kind(&value.to_string()) {
-                        out.kind = Some(kind);
-                    } else {
-                        return Err(syn::Error::new(value.span(), "unknown shader kind"));
+            if input.peek(Ident) {
```

Comment:
Style nit: we can avoid the rightward drift here with `if !input.peek(Ident) { break; }` or similar.

Reply:
Fixed it by replacing the `while` condition to do the equivalent